### ISSUE TITLE
Warning message: `pj_obj_create: Cannot find proj.db`

### ISSUE THREAD

**Issue by jomey** [conda-forge/python-pdal-feedstock#108]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

The changes to setting the plugin path (#106) seem trigger an error warning when activating an environment. Using version 3.3.0 did not show any warnings.

The install command with warning
```
conda env create -n pdal python-pdal
```

Error message:
```
pj_obj_create: Cannot find proj.db
```

The install command with no warning
```
conda env create -n pdal python-pdal=3.3.0
```

Activate via:
```
conda activate pdal
```

### Installed packages

```shell
python-pdal
```

### Environment info

```shell
conda version : 23.7.4
python version : 3.8.17.final.0
platform : linux-64
channel URLs : https://conda.anaconda.org/conda-forge/linux-64
               https://conda.anaconda.org/conda-forge/noarch
               https://repo.anaconda.com/pkgs/main/linux-64
               https://repo.anaconda.com/pkgs/main/noarch
               https://repo.anaconda.com/pkgs/r/linux-64
               https://repo.anaconda.com/pkgs/r/noarch
```

**Comment by jomey** [comment]
When another package like `gdal` is added, the warning message changes to:
```
pj_obj_create: Open of /data/conda/envs/pdal/share/proj failed
```

**Comment by floriandeboissieu** [comment]
I think it comes from the python-pdal activate script calling `python -m pdal` before proj paths are set.

It is solved be solved by commit 373d02b

**Comment by hobu** [comment]
Please reopen if this is still an issue with a fresh environment.
